core/application/src/env.rs

```diff
@@ -351,6 +352,25 @@ impl ApplicationEnv {
                 ),
             );
 
+            let total_intervals = if genesis.total_intervals == 0 {
+                1
+            } else {
+                genesis.total_intervals
+            };
+
+            time_interval_table.insert(
+                0,
+                (genesis.epoch_time + genesis.epoch_start)
```

Comment:
in line 762 of `epoch_change.rs`, the time interval is calculated by dividing `new_epoch_end_timestamp - old_epoch_end_timstamp` (or `epoch_time`) by the number of total intervals. Why are we adding the epoch start time here?

Reply:
This one is an oversight. Will fix.